bindings/matrix-sdk-ffi/src/timeline/configuration.rs

```diff
@@ -82,4 +139,26 @@ pub struct TimelineConfiguration {
 
     /// How often to insert date dividers
     pub date_divider_mode: DateDividerMode,
+
+    /// Should the read receipts and read markers be tracked for the timeline
+    /// items in this instance?
+    ///
+    /// As this has a non negligible performance impact, make sure to enable it
+    /// only when you need it.
+    pub track_read_receipts: bool,
+}
+
+/// Create a configuration for a live timeline, that will react to sync events
+/// in live.
+///
+/// Note: as of 2025-02-27, doesn't include the UTD hook.
+#[matrix_sdk_ffi_macros::export]
+pub fn live_timeline_configuration(internal_id_prefix: Option<String>) -> TimelineConfiguration {
```

Comment:
I'm not sure this brings much value tbh, I thought we were moving away fom multiple ways of creating timelines which this sort of diverges from.

Reply:
It's a way to create a timeline *configuration*, not a timeline. But maybe too early indeed; I'll keep that for when we get rid of the `init_timeline` method.